Tasks/Maksim/Task3.1/query06.sql

```diff
@@ -0,0 +1,14 @@
+--Query  6
+SELECT TOP (10)
+       SOD.SalesOrderID,
+       SUM(SOD.UnitPrice) TotalPrice
```

Comment:
LineTotal should be used instead to add Discount and Qnt of product to Total price calculation :)